src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --color: #00acdc;
+}
+
+.menu {
+  max-width: 1200px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.menu__logo {
+  display: inline-block;
+  padding: 10px 0;
```

Comment:
It is not quite right to center the element through the padding, if the height of the header changes, then you will need to change the padding. You have already set a property on the parent flex element that centers all children along the Y axis (align-items:center)